docs/reference/guides/plexguide.md

```diff
@@ -32,36 +32,49 @@ AND
 4. If you are not using encryption - Remove gcrypt: and tcrypt: if they exist.
 
 Final should look like:
-'
+```
 [google]
 type = union
 remotes = gcrypt: tcrypt:
```

Comment:
should this be `gdrive: tdrive:`?  The line above says to remove them.

Reply:
edits made